docusaurus/docs-internals/editor-controls/data-controls/internalLink.md

```diff
@@ -0,0 +1,182 @@
+
+![Internal Link Search](/img/data-controls/internalLinkSearch.png)
+
+Example of the internal link result:
+
+![Internal Link Result](/img/data-controls/internalLinkResult.png)
+
+### Parameters
+
+| Name               | Type         |    Default    | Description                                                                                                                       |
+|:-------------------|:-------------|:-------------:|:----------------------------------------------------------------------------------------------------------------------------------|
+| `id`               | `string`     |       -       | The identifier of the key where the internalLink will save your data                                                              |
+| `type`             | `string`     |       -       | Type should be `"internalLink"` to use this control                                                                               |
+| `placeholder?`     | `string`     |     -         | The placeholder text displayed in the input field.                                                                                |
+| `label?`           | `string`     |       -       | The label displayed on the left side of the control                                                                               |
+| `position?`        | `number`     |       -       | The position of the control in toolbar                                                                                            |
+| `devices?`         | `"all"`   \  | `"desktop"` \ | `"responsive"`  | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`        | `boolean`    |    `false`    | Configure the condition under which the control is disabled or enabled.                                                           |
+| `config?.size`     | `"short"`  \ | `"medium"` \  | `"large"` \| `"auto"`  |   `"medium"`   | A string that specifies the size of the input field.                                                                                                                                                                                                                  |
+| `config?.helper`   | `string`     |       -       | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears. |
+| `config?.sourceLabel`   | `string`     |   `"Type"`    | The label for the first select is displayed on the left side of the control.                                                              |
+
+### Return value
+
+Returns an object with the following values:
+
+```js
+{
+  value: string;
+  title: string;
+  source: string;
+}
+```
+
+`value`: A string that holds the actual data or content. <br/>
```

Comment:
value: the actual selected value
title: the label of the selected value
source: /* here describe that the source is the collection from where we try to find a link */